droidlet/dashboard/web/src/components/Interact/InteractApp.js

```diff
@@ -59,6 +63,9 @@ class InteractApp extends Component {
   }
 
   goToMessage() {
+    // Send a message to the parent iframe for analytics logging
+    window.parent.postMessage(JSON.stringify({ msg: "goToMessaage" }), "*");
```

Comment:
nit: goToMessaage -> goToMessage

Reply:
thanks for catching this, it may have been screwing up analytics a bit!